### ISSUE TITLE
Sway workspace, add class for focused output and focused workspace on all output

### ISSUE THREAD

**Issue by kuon** [Alexays/Waybar#1231]
It would be nice to be able to style the focused workspace on the non focused output.

For example, I have 3 output/displays. On the right one, I have workspace 4 and 5. When I have my mouse on the right display and workspace 5 is visible, the `.focused` class is applied to workspace 5's button. But if I move my mouse to my center display, no workspace have the `.focused` class, and I do not know which workspace on my right display is currently displayed.

The ideal solution would be to add the state (focused) to the waybar and have a displayed (or visible) state.

To summarize:

- Add `#waybar.focused` (or similar) for the focused output waybar styling (allowing to gray out non focused display/output).
- Add `button.displayed` when the workspace of the button is displayed.

We could remove `button.focused` because `#waybar.focused button.displayed` would replaced it, but maybe keep it for backward compatibility? (and it would be `#waybar.focused button.displayed.focused`)

**Comment by roberti42** [comment]
I like it.

**Comment by JohnTheCoolingFan** [comment]
You can use `.visible`

**Comment by roberti42** [comment]
Thank you @JohnTheCoolingFan, this is exactly what I was looking for.

I used it in conjunction with .focused to achieve what I wanted, here my config:
```css
#workspaces button.visible {
    border-top: 2px solid gray;
}

#workspaces button.focused {
    color: white;
    background: transparent;
    border-top: 2px solid white;
}
```

**Comment by rickalex21** [comment]
Is there a way to hide or style the unfocused bar? I'm using river. There is a
hide option.

Something like this:

```css
window#waybar.notfocused #window {  
 background:none;  
}  
```

**Comment by cvincent** [comment]
I would also be interested in the original ask on this issue, which is some top-level class to indicate whether the bar's output is the focused output. This would be really useful for a lot of use cases I think. As far I know, GTK CSS doesn't give us any way to style things based on the presence of a child with some class, or I could do something like (fake code):

```css
window#waybar:has-child(#workspaces button.visible) {
  /* styles for focused output bar here */
}
```

**Comment by Alexays** [comment]
Keeping open for the remaining ask. The per-workspace part is covered by the existing `button.visible` class (visible-but-not-focused workspace on any output). What is still missing is a top-level indication of which **output/bar** is focused — e.g. a `focused`/`active` class on `window#waybar` for the bar on the currently-focused output — so unfocused-output bars can be styled/grayed. That top-level focused-output class is the open feature here. 🙏
